server/Models/SurrenderApplication.js

```diff
@@ -0,0 +1,49 @@
+import pg from "pg"
+import AdoptablePet from "./AdoptablePet.js";
+import PetType from "./PetType.js"
+
+const pool = new pg.Pool({
+    connectionString: "postgres://postgres:password@localhost:5432/pets-app"
+})
+
+class SurrenderApplication{
+  constructor({id = null, name, phoneNumber, phone_number, email, petName, petAge,    petType, petImage, vaccinationStatus = "No"}){
+    //surrender_application fields
+    this.id = id;
+    this.name = name;
+    this.phoneNumber = phone_number || phoneNumber;
+    this.email = email;
+    this.status = "Pending";
+    //This needs to be set at record insert time, after the adoptable_pet is created
+    this.adoptable_pet_id = null;
+    //adoptable_pets fields
+    this.petName = petName;
+    this.img_url = petImage;
+    this.age = petAge;
+    this.pet_type = petType === "fish"? petType: petType + "s";
+    this.vaccination_status = vaccinationStatus || vaccination_status;
+    this.adoption_story = "Not available";
+  }
```

Comment:
We can remove the information that pertains to a pet.  We only need to track what will be stored on the surrender_applications table

Reply:
Kerrin had the same comment, but this data relates PetType, which is needed so we can get its ID so we can create the AdptablePet and subsequently create the SurrenderApplicaiton. I spent several hours refactoring this for Kerrin. 
So let me ask you the same questions I asked her:
If we can access any table in the db from any model, why wouldn't we?
I'm using the Models of the other classes within this Model to access their related tables. If you want my router to hit each table individually, we might as well have a separate router for each table, and for that matter, as a separate request from the front end for each router involved.